src/Components/Body/SortPanel/SortPanel.tsx

```diff
@@ -0,0 +1,18 @@
+import './SortPanel.scss'
+
+const sortOptions = ['RELEASE DATE', 'TITLE'];
+
+function SortPanel() {
+  return(<div className='sort-panel'>
+      <label>SORT BY</label>
+      <select>
+        {
+          sortOptions.map((i) =>
+            <option key={i}>{i}</option>
```

Comment:
Please doesn't create habit to use index as key. Value would be better in the case or you could use stricture a {name: ..., id: ... }